index.bs

```diff
@@ -1674,64 +1668,80 @@ should use {{USVString}}. [[HTML]]
-are mutable (and there is no way to make them immutable), which comes with a host of attendant
-problems.
+Don't use the JavaScript <code highlight="js">Date</code> class for representing
+specific date-time values.
+<code highlight="js">Date</code> objects are mutable (may have their value changed),
+and there is no way to make them immutable.
 
 <div class="note">
-    For more background on why <code highlight="js">Date</code> must not be used, see the following:
+    For more background on why <code highlight="js">Date</code> must not be used,
+    see the following:
 
     * <a href="https://esdiscuss.org/topic/frozen-date-objects">Frozen date objects?</a> on
         es-discuss
     * <a href="https://www.w3.org/Bugs/Public/show_bug.cgi?id=22824">Remove Date from Web IDL</a>
         on the Web IDL Bugzilla
 </div>
 
-While in theory time is monotonically increasing,
-values like {{DOMTimeStamp}} that represent time since 1970 are derived from the system's clock,
-which may sometimes move backwards
-(for example, from NTP or manual adjustment),
-causing the timestamp values to decrease over time.
-They may also remain the same due to the limitation of millisecond resolution.
-Thus, for time stamps that do not need to correspond to an absolute time,
-consider using {{DOMHighResTimeStamp}},
-which provides monotonically non-decreasing sub-millisecond timestamps
-that are comparable within a single <a>browsing context</a> or web worker.
+{{DOMTimeStamp}} values represent
+the number of milliseconds since 1970-01-01T00:00:00Z.
+They are derived from the system's clock,
+which may be changed by the user,
+or automatically for things like time zone updates
+or daylight savings.
+This means that, in practice, a {{DOMTimeStamp}}
+created later in real time
+may have a lower value than a {{DOMTimeStamp}} created earlier.
+Also, if two {{DOMTimeStamp}}s are created within the same millisecond,
+there's no way to know which was created earlier.
+
+So, if a date-time is only used to compare timing of events,
+and not to show a formatted date-time value to the user,
+consider using {{DOMHighResTimeStamp}} instead.
 See [[!HIGHRES-TIME]] for more details.
 
 <h3 id="error-types">Use Error or DOMException for errors</h3>
 
-Errors in web APIs should be represented as
-ECMAScript error objects (e.g., {{Error}})
+Represent errors in web APIs as ECMAScript error objects (e.g., {{Error}})
 or as {{DOMException}}.
-There was at one point a trend to use {{DOMError}} when
-objects had a property representing an error.
-However, we no longer believe there was value in this split,
-and therefore suggest that ECMAScript error objects (e.g., {{TypeError}})
-or {{DOMException}}
-should be used for errors,
+
+Issue: a negative example of one of the "older APIs" would be helpful.
+
+Some older APIs used {{DOMError}} when objects had a property representing an error.
```

Comment:
This is dead enough to a point that we could just remove this part and simply suggest DOMException.

Reply:
```suggestion
```